### PR TITLE
Fix shadowed and unused method

### PR DESCRIPTION
**Pull request by piotr-roslaniec**
**Type of PR:**
- [ ] Bugfix
- [ ] Feature
- [ ] Documentation
- [x] Other

**Required reviews:** 
- [x] 1
- [ ] 2
- [ ] 3

**What this does:**
- Replaces duplicated logic in `Alice.create_policy()` with call to `super` method `BlockchainPolicyAuthor.create_policy()`.

**Issues fixed/closed:**
- Fixes: #2442

**Why it's needed:**
> Explain how this PR fits in the greater context of the NuCypher Network.
> E.g., if this PR address a `nucypher/productdev` issue, let reviewers know!

**Notes for reviewers:**
- I'm not sure whether this PR covers 100% of the #2442

### CODE REVIEW

File: nucypher/characters/lawful.py [hunk 1]
```diff
@@ -253,9 +253,8 @@ def create_policy(self, bob: "Bob", label: bytes, **policy_params):
 
         else:
             # Sample from blockchain PolicyManager
-            from nucypher.policy.policies import BlockchainPolicy
             payload.update(**policy_params)
-            policy = BlockchainPolicy(alice=self, **payload)
+            policy = super(Alice, self).create_policy(**payload)
```
Comment (KPrasch) [inline nucypher/characters/lawful.py:257]:
```suggestion
            policy = super().create_policy(**payload)
```

**Review by cygnusv — APPROVED** [review]
Looks good to me! Thanks @piotr-roslaniec :)

**Review by derekpierre — APPROVED** [review]
🎸

**Review by KPrasch — APPROVED** [review]
🤠

**Pull request merged**